run_common.go

```diff
@@ -642,7 +643,15 @@ func runUsingRuntime(options RunOptions, configureNetwork bool, moreCreateArgs [
 		}
 	}()
 	signal.Notify(interrupted, syscall.SIGHUP, syscall.SIGINT, syscall.SIGTERM)
+	iter := 0
 	for {
+		iter = iter + 1
+		exponential := int(math.Min((math.Pow(2, float64(iter)) * 50), 2000))
+		// Reached the max delay time reset counter to 0
+		// and start exponential delay again.
+		if exponential == 2000 {
```

Comment:
I am not sure of this code here, should we just stay at 2 seconds once it gets here rather then speeding up again?